core/fed_avg_optimizer.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, initialization_payload, dataset_manager):
 		max_rounds(int): how many rounds of fed learning this optimizer wants to do
 		"""
 		logging.info("Setting up Optimizer...")
+		# initialization_payload = initialization_payload.get_tx()
```

Comment:
Did you mean to leave this commented out?

Reply:
Yep -this occurs only when the initialization_payload ends up being a Transaction object.

Reply:
Just going to delete this actually.